src/league.js

```diff
@@ -164,7 +176,7 @@ league_attributes = {
             _.each(roster.teams, function(team) {
                 _.each(team.players, function(teamPlayer) {
                     var player = self.getPlayer(teamPlayer.username);
-                    player.isCaptain = player.is_captain;
+                    player.isCaptain = teamPlayer.is_captain;
```

Comment:
Yes! There is the bug. I didn't look hard, but I noticed this bug when I first wrote the code. Nice catch!